doc/examples/data/plot_specific.py

```diff
@@ -55,9 +55,52 @@
 
 fig, axes = plt.subplots(4, 5, figsize=(20, 20))
 ax = axes.ravel()
-images = data.lfw_subset()
+lfs_images = data.lfw_subset()
 for i in range(20):
-    ax[i].imshow(images[90+i], cmap=plt.cm.gray)
-    ax[i].axis('off')
+    ax[i].imshow(lfs_images[90 + i], cmap=plt.cm.gray)
+    ax[i].axis("off")
 fig.tight_layout()
 plt.show()
+
+
+######################################################################
+# Thumbnail image for the gallery
+
+# sphinx_gallery_thumbnail_number = -1
+from matplotlib.offsetbox import AnchoredText
+
+ax1 = plt.subplot(2, 2, 1)
+ax1.imshow(cylce_images[0])
+ax1.add_artist(
+    AnchoredText(
+        "Stereo",
+        prop=dict(size=20),
+        frameon=True,
+        borderpad=0,
+        loc="upper left",
+    )
+)
```

Comment:
Just curious, why did you choose to add a description here but not on the LFW images?

Reply:
Because I liked to include multiple LFW images, and with these very small LFW images, the title would overlap the image too much. I think like this it looks good, with a title the thumbnail would be a bit too cluttered.